courses/migrations/0033_populate_letter_grades.py

```diff
@@ -4,16 +4,22 @@
 import pytz
 from django.db import migrations
 
+from courses.models import Program
 from courses.utils import convert_to_letter
 
 
 def populate_letter_grade(apps, schema_editor):
     """Populate the certificate_available_date from course run's end_date"""
     CourseRunGrade = apps.get_model("courses", "CourseRunGrade")
+
+    dedp_program = Program.objects.filter(readable_id="program-v1:MITx+DEDP").get()
```

Comment:
Will this break locally if a developer does not have this program configured?  If so, this might be difficult for someone to recognize when first setting up their environment.

Reply:
good call. I will update to use `.first()` so it will not throw error if no such program locally